Lib/glyphsLib/builder/custom_params.py

```diff
@@ -529,6 +530,43 @@ def to_glyphs_gasp_table(value):
 )
 
 
+# convert Glyphs' meta Table to UFO openTypeMeta
+def to_ufo_meta_table(value):
+    meta = {}
+    # In:  {data = "de-Latn"; tag = dlng; }, {data = "sr-Cyrl"; tag = slng; }
+    # Out: { "dlng": [ "de-Latn" ], "slng": [ "sr-Cyrl" ] }
```

Comment:
fonttools and UFO spec currently store meta data as a dictionary thus disallowing entries with duplicate keys, whereas Glyphs.app follows OT spec and allows that:

https://github.com/fonttools/fonttools/issues/2345
https://github.com/unified-font-object/ufo-spec/issues/187

Maybe that will change if somebody bothers to change that, but for now that's how it is, I suggest you log a warning in the case of duplicate keys. Right now you are taking the last, either that or taking first is fine, both are equally "wrong" anyway.